### ISSUE TITLE
Vercel deploy error

### ISSUE THREAD

**Issue by zntb** [bradtraversy/prostore#2]
The app cannot be installed on Vercel with a Hobby account because it exceeds 1Mb.
![Vercel error](https://github.com/user-attachments/assets/6805fd44-a698-4133-b6d8-a7f4693baf70)

**Comment by kevinsherman** [comment]
I'm running into the same issue... is there any way to slim-down the imports?  
![image](https://github.com/user-attachments/assets/c0abd74d-7252-408e-8153-1642c342297e)

**Comment by zntb** [comment]
Unfortunately, I haven't found a way yet. I installed it on Netlify, but the site crashed and this error appears:
![Netlify](https://github.com/user-attachments/assets/afa44b36-8e68-410b-8e77-19063ccc5615)

**Comment by bradtraversy** [comment]
oh I wasn't aware of that. I will look into this and  report back. Thanks

**Comment by bradtraversy** [comment]
Alright, so we were able to deploy to the hobby tier by removing the bcrypt-ts-edge package and using the Web Crypto API to encrypt passwords. I created a Gist with the exact steps on how to do this. We will also be adding a branch to this repo for people that want to deploy to the hobby tier.

[https://gist.github.com/bradtraversy/b83e96d027f51485d9385e557855db62](https://gist.github.com/bradtraversy/b83e96d027f51485d9385e557855db62)

Here is the Gist. It is pretty straightforward. I will also be adding a video at the end of the course with this solution.
